### ISSUE TITLE
what is the better/best optimization policy(xyz_gradient_accum)?    (has detail desciption)

### ISSUE THREAD

**Issue by yuedajiong** [graphdeco-inria/gaussian-splatting#370]
now:

for iteration:
    ...
    loss.backward()     #gradients generated !!!
    ...
    add_densification_stats()  #gradients accumulated !!!
    ...
    if iteration > densify_from_iter and iteration % densification_interval == 0:
        densify_and_prune()  ->  densify_and_clone() | densify_and_split() -> densification_postfix()   #gradients zero-ed !!!
    optimizer.step()  #gradients used, every epoch.

question:
    how to get a better 'gradients accumulation/re-zero' hyperparameter?   
        fixed like the paper: epochs//fixed-number  
        OR
        refer to the number of images?  1x 2x 3x Nx times of the number of images?

**Comment by yuedajiong** [comment]
not an issue

**Comment by yuedajiong** [comment]
not an issue
